SlackHistoryViewer.Helpers/SlackHistoryViewer.Helpers.csproj

```diff
@@ -1,7 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard1.4</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\SlackHistoryViewer.Database\SlackHistoryViewer.Database.csproj" />
+    <ProjectReference Include="..\SlackHistoryViewer.Slack\SlackHistoryViewer.Slack.csproj" />
```

Comment:
Those project don't need to be in the helpers. If you need this, then you have to refactor :)